lib/plain/entities/resource-type.ts

```diff
@@ -70,6 +74,24 @@ export type ResourceTypePlainClientAPI = {
    */
   delete(params: OptionalDefaults<GetResourceTypeParams>): Promise<any>
 
+  /*
+   * Fetch all Resource Types for an environment
+   * @param params entity IDs to identify the Resource Type
+   * @params Optional query params for cursor pagination
+   * @returns all Resource Types based on the last NER app installed in the environment
+   * @throws if the request fails, or no Resource Type is found
+   * @example
+   * ```javascript
+   * const resourceTypes = await client.resourceType.getForEnvironment({
+   *   spaceId: '<space_id>',
+   *   environmentId: '<environment_id>',
+   * });
+   * ```
+   */
+  getForEnvironment(
+    params: OptionalDefaults<GetSpaceEnvironmentParams> & { query?: BasicCursorPaginationOptions }
```

Comment:
query is optional here but required in `lib/adapters/REST/endpoints/resource-type.ts`